server/apps/main/templates/main/moderation_page.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <title>AutSPACEs Moderation Page</title>
+    <style>
+        body {
+            background: #E8F3FC;
+        }
+        .my-custom-card{
+            background:#C4C4C4;
+			height: 699px;
+
+        }
+        .goals{
+            background:#EBF1F5;
+        
+        }
+        .navbar{
+            background: rgba(149, 239, 234, 0.42);
+			font-size: 24px;
+
+        }
+		form{
+            text-align: center;
+            font-size: x-large;
+        }
+		.mysmallCcards{
+			background: #00B2FF;
+			
+		}
+        .btn-colour{
+            height: 80.84px;
+            width: 200px;
+		}
+		.btn-colour:hover{
+            background: #7dce1b;
+            height: 80.84px;
+            width: 200px;
+        
+        }
+        
+    </style>
+</head>
+
+<body>
+	<nav class="navbar">
+		<a class="navbar-brand" href="#" </a>
+
+			 <a style="color:#000000;"href="#about">About</a>      
+			 <a style="color:#000000;"href="#share">Share</a>      
+			 <a style="color:#000000;"href="#login">Login</a>

```

Comment:
Could you add the navigation items similar to how you're adding them in PR #116 ? 

They don't need to be functional. They just need to exist so that there's something to work with. (See issue #118 for additional task details.)

Reply:
@jhlink I have the requested changes and updated the code :)

Reply:
I'll take a look at the outstanding PRs during my lunch today. I appreciate the reminder!